content/en/platform/corda/5.2/developing-applications/notaries/_index.md

```diff
@@ -1,90 +1,36 @@
+A notary service is formed by one or more notary workers that together form a notary cluster. The cluster’s signature is obtained once it verifies that a proposed transaction’s input states have not already been consumed by a prior transaction. Upon determining this, the notary cluster will either:
 
-* [Network Member Roles](#network-member-roles)
-* [Notary Plugin CorDapps](#notary-plugin-cordapps)
+* Sign the transaction in cases where all input states are found to be unique.
+* Reject the transaction and flag that a double-spend attempt has occurred in cases where any of the input states are identical to those already encountered in a previous transaction.
 
-{{< note >}}
-Notary virtual nodes use an additional “uniqueness” database for capturing state data for double-spend prevention. This is similar to the existing “crypto” and “vault” databases. Currently, when auto-provisioning virtual node databases, a uniqueness database is always provisioned, regardless of whether it is a notary virtual node or not. This will be addressed in a future release.
-{{< /note >}}
+Every state has an appointed notary cluster, so the cluster only notarizes a transaction if it is the appointed notary cluster of all of the transaction’s input states. A network can have several notary clusters, all running different consensus algorithms. 
 
-## Network Member Roles
+## Multiple Notaries
 
-With the introduction of the {{< tooltip >}}UTXO{{< /tooltip >}} ledger and notaries, Network Operators must be aware of the different roles that exist on a network. Network participants can take one of three roles:
+Each Corda network can have multiple notary clusters. This has several benefits:
 
-* **Application:** Members which run a {{< tooltip >}}CPI{{< /tooltip >}} containing a {{< tooltip >}}CPB{{< /tooltip >}} which provides a {{< tooltip >}}CorDapp{{< /tooltip >}} to run on the network. Most members fulfil this role, and would be our archetypical “Alice” or “Bob” nodes.
-* **Notary:** Members which run a CPI containing a CPB which provide a notarization service to the network. These are effectively representatives of a notary service, similar to how notary workers represented a notary service in a Corda 4 high-availability notary setup.
-* **MGM:** The Membership Group Manager. In the context of notary functionality, there is nothing special or different about the {{< tooltip >}}MGM{{< /tooltip >}}. The MGM virtual node is deployed as part of your standard network setup.
+* **Choice of protocol:** Nodes can choose the preferred notary cluster on a per-transaction basis.
+* **Load balancing:** Spreading the transaction load over multiple notary clusters allows higher transaction throughput for the platform overall.
+* **Low latency:** Latency can be minimized by choosing a notary cluster physically closer to the transacting parties.
 
-## Notary Plugin CorDapps
+## Changing Notaries
```

Comment:
I don't think changing notaries is implemented on Corda 5 as yet. Is that correct @rick-r3 ?

Reply:
The text does say "This has not been implemented for this version of Corda."
Perhaps it would be better to remove the section rather than highlighting something that's not available?